### PR TITLE
Adding delegate factory: Listen to events to create delegates

### PR DESCRIPTION
**Pull request by rafinskipg**

### CODE REVIEW

File: subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts [hunk 1]
```diff
@@ -1,85 +1,47 @@
...
-  //   delegate.id,
-  //   sender.toHexString()
-  // );
-  // delegateVPChange.save();
+  // Track the change of MKR locked in chief for the user
+  const chiefLockedMKRChange = createChiefLockedMKRChange(
+    event,
+    voter.mkrLockedInChiefRaw,
+    voter.mkrLockedInChiefRaw.plus(amount),
+    voter.id
```
Comment (adamgoth) [inline subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts]: I recently added the sender of the tx to this function. We need to know that in order to show who delegated the MKR.

File: subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts [hunk 2]
```diff
@@ -121,20 +88,20 @@ export function handleEtch(event: Etch): void {
   const to = event.transaction.to;
   // Check if txn is not directly to Chief, it's either to vote delegate or multi-sig + delegate
   if (to && to != event.address) {
-    const fromAdmin = DelegateAdmin.load(sender);
+    const fromAdmin = Voter.load(sender);
     if (!fromAdmin) {
```
Comment (adamgoth) [inline subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts]: This looks like it was checking if someone was a delegate owner or not. Now the logic is just checking if they are a voter (from what it seems).

File: subgraphs/maker-governance/schema.graphql [hunk 3]
```diff
@@ -148,15 +170,13 @@ type Delegation @entity {
   amount: BigInt!
 }
 
-type DelegateVotingPowerChange @entity(immutable: true) {
+type ChiefLockedMKRChange @entity(immutable: true) {
```
Comment (adamgoth) [inline subgraphs/maker-governance/schema.graphql]: I think `ExecutiveVotingPowerChange` sounds a bit cleaner and more descriptive. `ChiefLockedMKRChange` feels awkward and un-intuitive. Wdyt?

Comment (rafinskipg) [reply subgraphs/maker-governance/schema.graphql]: nice yeah

File: subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts (hunk 2 above)
Comment (rafinskipg) [reply subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts]: yeah, so maybe i should check if the voter is a delegate? I'll try that. 

also i'm not super sure if delegateadmin always was a delegate admin before

File: subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts (hunk 1 above)
Comment (rafinskipg) [reply subgraphs/maker-governance/protocols/maker-governance/src/ds-chief.ts]: but for that we have the MKR delegation history , Delegation, is there any case where we want to see also when the total chief MKR increased who was the origin of it?

**Pull request merged**